### PR TITLE
[MOBDW-82] Internationalization Added to Menu Bar

### PR DESCRIPTION
**Pull request by BrianCorbin**
Soundtrack of this PR: [Deja Vu](https://www.youtube.com/watch?v=dv13gl0a-FA)

### Motivation

i18next is a pain, and for some reason the app was no longer working on Mac's when performing yarn start OR yarn package.

### In this PR

Reverting to the last stable main section and reimplemented, is currently stable.

### Testing
Test Suites: 25 passed, 25 total
Tests:       108 passed, 108 total
Snapshots:   8 passed, 8 total
Time:        29.08 s
Ran all test suites.
✨  Done in 32.22s.

### CODE REVIEW

File: app/menuFactory.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Menu } from 'electron';
+
+import config from '../configs/app.config';
+import darwinTemplate from './menus/darwin-menu';
+import otherTemplate from './menus/other-menu';
```
Comment (davidyhall) [inline app/menuFactory.js:5]: Let's keep filenames camelCase

File: app/menuFactory.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Menu } from 'electron';
```
Comment (davidyhall) [inline app/menuFactory.js:1]: Let's use Typescript

File: app/i18n.ts [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
-  // init i18next
-  .init({
+  // .use(i18nextBackend)
+  .use(initReactI18next);
+
+if (!i18n.isInitialized) {
+  i18n.init({
     debug: true,
-    fallbackLng: 'enUS',
+    fallbackLng: 'en',
```
Comment (davidyhall) [inline app/i18n.ts:16]: I think long term their may be different en versions. also fine with renaming to `'en-US'`

File: app/i18next.config.client.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+const i18nextOptions = {
+  fallbackLng: config.fallbackLng,
+  interpolation: {
+    escapeValue: false,
+  },
+  lng: 'en',
+  react: {
+    wait: false,
+  },
+  saveMissing: true,
```
Comment (davidyhall) [inline app/i18next.config.client.js:15]: We won't need saveMissing

File: app/i18next.config.client.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  whitelist: config.languages,
```
Comment (davidyhall) [inline app/i18next.config.client.js:16]: Double check this key. I think it may have been deprecated for `supportedLngs` -- I would check all the keys tbh.

**Review by davidyhall — CHANGES_REQUESTED** [review]
Great start!

I'm not seeing the menu correctly build when running `yarn package` -- it looks like it's generating the default menu.

**Review by davidyhall — CHANGES_REQUESTED** [review]
It looks like the menu is still not building correctly in `yarn package`

**Comment by BrianCorbin** [comment]
Unnecessary since all of these changes are captured in [PR#50](https://github.com/mobilecoinofficial/desktop-wallet/pull/50)

**Pull request closed without merging**